backend/FwLite/FwLiteProjectSync.Tests/SyncTests.cs

```diff
@@ -517,7 +534,10 @@ public async Task AddingASenseToAnEntryInEachProjectSyncsAcrossBoth()
                 .For(e => e.Senses).Exclude(s => s.Order)
                 .For(e => e.Senses).For(s => s.ExampleSentences).Exclude(s => s.Order)
                 .For(e => e.Components).Exclude(c => c.Id)
-                .For(e => e.ComplexForms).Exclude(c => c.Id));
+                .For(e => e.Components).Exclude(c => c.Order)
+                .For(e => e.ComplexForms).Exclude(c => c.Id)
+                .For(e => e.ComplexForms).Exclude(c => c.Order)
```

Comment:
this is getting silly, can you pull out a helper method and use that everywhere in this test class

Reply:
fixed this myself